modular_citadel/code/modules/reagents/reagent container/hypovial.dm

```diff
@@ -49,21 +50,35 @@
 		filling.color = mix_color_from_reagents(reagents.reagent_list)
 		add_overlay(filling)
 
+/obj/item/reagent_containers/glass/bottle/vial/tiny
+	name = "small hypovial"
+	//Shouldn't be possible to get this without adminbuse
```

Comment:
what is even the point of a 'tiny' vial then?

Reply:
The hypospray can't accept bottle/vial because otherwise it would accept all vials. Tiny is there for no purpose other than to provide a usable version of bottle/vial if at all necessary.

Reply:
if you have it only accept the small vial, then it'll be fine. that's why I separated the "normal" to a "small" I tested that ~

Reply:
It also has 1 and 2 unit injection options, which can be useful when spraying Sulfonal or Ketrazine (if that wasn't patched out within like two days) where more isn't necessary.